### DISCUSSION TITLE
Access to firestore-array in rowy-table

### DISCUSSION THREAD

**Discussion by webwurm** [AntlerVC/firetable#948] [Support / Q&A] [answered]
I have a short question:
I have a field "tags" which has a list of references in my firestore-documents, like so:
<img width="281" alt="Tag-List" src="https://user-images.githubusercontent.com/1053176/201985777-78812f9a-c566-4228-b0ef-7240a0a27261.png">

Can I display/access this in rowy somehow?
I would need that more often, since this is what FlutterFlow creates when you define a document-field and choose the option "Is a list"...

Thanks!

**Comment by BeeBombshell** [comment, marked as answer]
I believe this issue was resolved in the Discussion: https://github.com/rowyio/rowy/discussions/973. Therefore, closing this one...
